<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCategoryValue</class>
 <widget class="QWidget" name="FormCategoryValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>25</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ebrima</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="LabelPosition">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">Pos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LineEditTitle"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="CheckBoxVisible">
     <property name="text">
      <string>Visible</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonReverse">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonUp"/>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonDown"/>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonSelect"/>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonUnSelect"/>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonDelete"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
